doc/conf.py

```diff
@@ -35,7 +37,35 @@
 release = xgboost.__version__
 
 
-def run_doxygen():
+# Witchcraft alert:
```

Comment:
took me a while to figure out that the existing code was broken after the CI revamp. Would be great if we could use a package manager like conda to handle R packages instead of spawning out a new CI workflow.